src/documentation/ToolkitDocumentation/CodeEmbed.tsx

```diff
@@ -68,36 +72,46 @@ const CodeEmbed = ({ code: codeWithImports }: CodeEmbedProps) => {
     <Box>
       <Box height={codeHeight} fontSize="md">
         <Code
-          // Shadow only on this one, not the pop-up.
-          boxShadow="rgba(0, 0, 0, 0.18) 0px 2px 6px;"
           onMouseEnter={handleMouseEnter}
           onMouseLeave={handleMouseLeave}
           concise={code}
           full={codeWithImports}
           position="absolute"
           ref={codeRef}
+          background={hovering || hoverInsertBtn ? "#E9F6F5" : "white"}
+          hovering={hovering || hoverInsertBtn}
+          hoverDragIcon={hoverDragIcon}
+          setHoverDragIcon={setHoverDragIcon}
         />
         {hovering && (
           <CodePopUp
-            height={codeHeight}
+            height={codePopUpHeight}
+            width={codeRef.current ? codeRef.current.offsetWidth : "unset"}
             setHovering={setHovering}
             concise={code}
             full={codeWithImports}
             codeRef={codeRef}
+            background={hovering || hoverInsertBtn ? "#E9F6F5" : "white"}
+            hoverDragIcon={hoverDragIcon}
+            setHoverDragIcon={setHoverDragIcon}
           />
         )}
       </Box>
-      <HStack spacing={3}>
+      <HStack spacing={3} mt="2px">
         <Button
+          onMouseEnter={() => setHoverInsertBtn(true)}
+          onMouseLeave={() => setHoverInsertBtn(false)}
           fontWeight="normal"
-          color="white"
-          borderColor="rgb(141, 141, 143)"
-          bgColor="rgb(141, 141, 143)"
+          color="gray.800"
+          border="none"
+          bgColor={hoverInsertBtn ? "#95D7CE" : "#CAEBE7"} //unlisted colors
           borderTopRadius="0"
-          borderBottomRadius="xl"
+          borderBottomRadius="lg"
+          ml={5}
           variant="ghost"
           size="sm"
           onClick={handleInsertCode}
+          rightIcon={<RiFileTransferLine />}
```

Comment:
Matt to look at icon options.

Reply:
Figured out they'd rotated a download icon and did likewise with a simpler remix one. It looks a little better, I think.